tests/phpunit/TestHelpers/GetDomHelperTrait.php

```diff
@@ -0,0 +1,31 @@
+<?php // phpcs:ignore WordPress.Files.FileName.NotHyphenatedLowercase -- underscore is for valid function name.
+/**
+ * A helper class to generate dom objects that match the shape of the
+ * object in the plugin that is used when running rules.
+ *
+ * @package Accessibility_Checker
+ */
+
+/**
+ * Gets dom objects through EDAC_Dom that implements the same interface SimpleHTMLDom.
+ */
+trait GetDomHelperTrait {
+
+	/**
+	 * Wrapper to generate dom objects that match the shape of the object in the plugin.
+	 *
+	 * @param string $html_string HTML string.
+	 * @return EDAC_Dom
+	 */
+	public function get_DOM( string $html_string = '' ): EDAC_Dom { // phpcs:ignore WordPress.NamingConventions -- This is a helper function.
+		return new EDAC_Dom(
```

Comment:
I don't think this rule used the `EDAC_Dom` class. Is there a reason to use this over using `str_get_html()`?

Reply:
The `str_get_html()` likely has the same string generated as what this is for. I used this though because I expected that it would be best to get the html string in exactly the same way as it would be passed to the rule function from in the plugin.

One way in the plugin, one way in the tests and they are the same is the general idea. It wouldn't make a huge amount of difference if I dropped this in favour of just calling `str_get_html()` though and I can do that going forward if you feel strongly about simplifying in that way.